playbooks/openstack/advanced-configuration.md

```diff
@@ -327,17 +327,34 @@ The `openshift_openstack_required_packages` variable also provides a list of the
 prerequisite packages to be installed before to deploy an OpenShift cluster.
 Those are ignored though, if the `manage_packages: False`.
 
-## Multi-master configuration
+## API and Router Load Balancing
+
+If you run multiple master or infra nodes, you want to put a load balancer in
+front of them.  We can configure an Octavia or Neutron LBaaS V2 load balancer
+automaticaly.
+
+All you need to do is put this in your `inventory/group_vars/all.yml`:
+
+    openshift_openstack_use_lbaas_load_balancer: true
```

Comment:
I was about to leave a separate comment asking if we want to put this option in the sample inventory file; then I remember you saying that you'd like to start emptying that out and simply have the possible parameters listed in the documentation, correct?

If so, I think we may want to organize that documentation a bit differently, so it's easier for someone to simply skim the list of possible parameters to find the ones that they care about. I can take a shot at that if you'd like.

Reply:
Yeah, I'd like to prune some of the less important and space-consuming options out.

But this one is actually quite important, I think. I'll add it and we can clean it all up in another patch.